web/client/epics/__tests__/styleeditor-test.js

```diff
@@ -1475,5 +1475,48 @@ describe('Test styleeditor epics, with mock axios', () => {
             state);
 
     });
+    it('test toggleStyleEditorEpic enabled to true and layer url is array', (done) => {
 
+        const state = {
+            layers: {
+                flat: [
+                    {
+                        id: 'layerId',
+                        name: 'layerName',
+                        url: ['/geoserver1/', '/geoserver2/', '/geoserver3/']
+                    }
+                ],
+                selected: [
+                    'layerId'
+                ]
+            }
+        };
+        const NUMBER_OF_ACTIONS = 1;
+
+        const results = (actions) => {
+            expect(actions.length).toBe(NUMBER_OF_ACTIONS);
+            try {
+                actions.map((action) => {
+                    switch (action.type) {
+                    case LOADING_STYLE:
+                        expect(action.status).toBe('global');
```

Comment:
this is a false positive, i do not think this is actually testing new added code.

Reply:
Adjusted test to better check the actual value in https://github.com/geosolutions-it/MapStore2/pull/10004/commits/22f0d9ba262e3352d0572dcd6c748162626449d7